### PR TITLE
MINOR: Fix more places where the version should be bumped from 2.1.0 …

### PR DESCRIPTION
**Pull request by cmccabe**

### CODE REVIEW

**Comment by hachikuji** [comment]
Do we have a doc somewhere which lists everywhere the version needs to be updated?

File: kafka-merge-pr.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
 
 DEV_BRANCH_NAME = "trunk"
 
-DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.1.0")
+DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.1.1")
```
Comment (ijuma) [inline kafka-merge-pr.py:73]: This one is not needed since this tool should always be executed from trunk.

**Comment by ijuma** [comment]
@hachikuji https://github.com/apache/kafka/pull/6226/files#diff-503f218d646c10f484fdc9d6315bf2e3L22

File: kafka-merge-pr.py (hunk 1 above)
Comment (cmccabe) [reply kafka-merge-pr.py:73]: I agree it doesn't need to be, but it seems like it should be for consistency, and to avoid confusion.

**Review by ijuma — APPROVED** [review]
LGTM

**Pull request merged**